tensorboard/plugins/pr_curve/summary.py

```diff
@@ -164,6 +164,98 @@ def op(
         description,
         collections)
 
+
+def streaming_op(tag, labels, predictions, num_thresholds=200, weights=None,
```

Comment:
nit: Could we reorganize the arguments to be one per row?

Reply:
Done.